apps/chat/src/components/Chat/Publish/PublishWizard.tsx

```diff
@@ -68,6 +69,11 @@ import isEqual from 'lodash-es/isEqual';
 import startCase from 'lodash-es/startCase';
 import toLower from 'lodash-es/toLower';
 
+const modalDismissProps = {
+  outsidePress: false,
+  escapeKey: false,
+} as UseDismissProps;
```

Comment:
I suggest to put two constants ALLOW_CLICK_OUTSIDE and DISALLOW_CLICK_OUTSIDE in some constants/dialogs.ts and use everywhere